app/models/user.rb

```diff
@@ -12,13 +12,36 @@
 #
 
 class User < ApplicationRecord
+  attr_accessor :password
+
   has_many :posts, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_and_belongs_to_many :books, join_table: :book_users, dependent: :destroy, class_name: 'Book'
 
   has_one :avatar, as: :imageable, class_name: 'Image', dependent: :destroy
   has_one :billing_info, dependent: :destroy
 
+
   validates :username, uniqueness: true, presence: true
-  validates :password, length: { minimum: 6 }, presence: true
+  validates :password, length: { minimum: 6 }, presence: :create
```

Comment:
@SemyzSerhiy `presence: :create` is incorrect. Please change to `presence: true`